### ISSUE TITLE
Return helpful response if any required header is missing

### ISSUE THREAD

**Issue by gr2m** [probot/adapter-aws-lambda-serverless#65]
follow up to https://github.com/probot/adapter-aws-lambda-serverless/pull/63

/cc @axel3rd  I think that would help saved you a lot of headache.

**Comment by axel3rd** [comment]
Yes 🤯 ^^, but could be part of #60

**Comment by gr2m** [comment]
yes I agree. I'm just not sure how pointing to https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs.html would be helpful?

**Comment by axel3rd** [comment]
> I'm just not sure how pointing to https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs.html would be helpful?

IMO, the CloudWatch logs is easy to access from Monitoring Tab (and not sure than [Lambda Insight](https://docs.aws.amazon.com/lambda/latest/dg/monitoring-insights.html) can be relevant here).

Using [Probot log](https://probot.github.io/docs/logging/) can do the job (`log.debug` for input event, `log.error` in the error catch).

**Comment by gr2m** [comment]
so in case of an error response, shall we include a JSON key such as 

```json
{
  "error": "...",
  "documentation_url": "https://docs.aws.amazon.com/lambda/latest/dg/monitoring-cloudwatchlogs.html"
}
```
